_docs/install.md

```diff
@@ -7,28 +7,33 @@ toc:
 nav: 1
 ---
 
-## Install
+Install
+-------
 
 You can install Pico either using a pre-bundled release or with composer. Pico is also available on [Packagist.org][] and may be included in other projects via `composer require picocms/pico`. Pico requires PHP 5.3+
 
-### Using a pre-bundled release
+### Using a pre-bundled release - for users
```

Comment:
No! The `composer` setup is not just for developers. There's no change necessary in this section...